content/Hardware/Generic/How-to-do-a-loopback-test.md

```diff
@@ -2,19 +2,17 @@
 title: "How to do a loopback test"
 ---
 
+A **loopback test** is a troubleshooting procedure to test the serial communication between the PC and Arduino board with a dedicated USB to Serial converter chip. It can be performed with the following boards:
 
-For Arduino boards with a dedicated USB to serial adapter chip (Uno, Mega, or Nano), a *loopback test* can be performed to test the serial communication between the board and your PC.
```

Comment:
We will have to test and confirm on the different versions of the board just to be sure it works.

Reply:
@jojo1212 I've tested the Rev3 SMD. I can look in the lab for the Mega variants.

Reply:
I couldn't find a Arduino Mega ADK Rev3 but otherwise I think we're covered.

Reply:
We will keep a note of this and whenever we will be able to get our hands on this board we will try it. I will check with David to see if he has any that can be borrowed.